package.json

```diff
@@ -345,7 +345,7 @@
         "react-tether": "^1.0.5",
         "react-textarea-autosize": "^8.5.3",
         "regenerator-runtime": "^0.13.2",
-        "sanitize-html": "^1.21.1",
+        "sanitize-html": "^2.12.1",
```

Comment:
only usage of this package is:
https://github.com/box/box-ui-elements/blob/fbe87d4078c5c7de954708978bc3642aea4d2001/src/features/message-center/components/templates/common/MessageTextContent.js#L4

seems safe to merge?
https://github.com/apostrophecms/sanitize-html/blob/main/CHANGELOG.md#200-2020-09-23